__tests__/app.test.js

```diff
@@ -61,6 +61,26 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 
+  test("200: Includes comment_count within the returned article object", () => {
```

Comment:
You already checked the other properties so you would have been okay just to check the comment_count key in this test.